pictures/tasks.py

```diff
@@ -41,6 +44,21 @@ def _process_picture(
         picture = utils.reconstruct(*picture)
         picture.delete()
 
+    if field:
+        app_label, model_name, _ = field.split(".")
+        sender = apps.get_model(app_label=app_label, model_name=model_name)
+    else:
+        sender = _process_picture
+
+    signals.process_picture_done.send(
+        sender=sender,
+        storage=storage.deconstruct(),
+        file_name=file_name,
+        new=new,
+        old=old,
+        field=field,
```

Comment:
I wonder if it would be cleaner to make the field the sender. In that case you wouldn't need to pass it twice.
```suggestion
```

However, you should add documentation on how to use the signal. Someting like:

```python

class MyModel:
    picture = PictureField()


@receiver(picture_process, sender=MyModel._meta.get_field("picture")
def fn():
    """Post processor"""

```